配置/默认/TCMMF411

```diff
@@ -0,0 +1,112 @@
+# Betaflight / STM32F411 (S411) 4.1.1 Nov 15 2019 / 12:55:17 (1e5e3d369) MSP API: 1.42
```

Comment:
Path and file name for this are wrong - it needs to be in `configs/default/<manufacturer id>-<board name>.config